logicrunner/goplugin/goplugin.go

```diff
@@ -29,7 +29,9 @@ import (
 	"time"
 
 	"github.com/insolar/insolar/logicrunner"
-	"github.com/insolar/insolar/logicrunner/goplugin/girpc"
+	"github.com/insolar/insolar/logicrunner/goplugin/rpctypes"
+	"github.com/insolar/insolar/messagerouter"
+	"github.com/insolar/insolar/network/host"
```

Comment:
could not import github.com/insolar/insolar/network/host (cannot find package "github.com/insolar/insolar/network/host" in any of:
	/tmp/tmp.9XXbyXvxsY/src/github.com/insolar/insolar/vendor/github.com/insolar/insolar/network/host (vendor tree)
	/usr/local/go/src/github.com/insolar/insolar/network/host (from $GOROOT)
	/tmp/tmp.9XXbyXvxsY/src/github.com/insolar/insolar/network/host (from $GOPATH))